test/rewrite_clj/node/coercer_test.cljc

```diff
@@ -66,7 +67,7 @@
   (testing "multi-line string newline variants are normalized"
     (let [s "hey\nyou\rover\r\nthere"
           n (node/coerce s)]
-      (is (= "hey\nyou\nover\nthere" (node/sexpr n))))))
```

Comment:
I changed the behavior here to always leave the original string untouched, which seems better to me in case you have literal strings with `"foo\r\n"` in your code, which is valid.

Reply:
Maybe, but we do we care?

Reply:
What I mean is, does rewrite-clj not do what it's supposed to do if we forget about this detail?

Reply:
Yes, I suppose, might not be worth worrying about.